### PR TITLE
adapt to ocis monorepo

### PR DESCRIPTION
**Pull request by micbar**
# Description

- Adapt the batch processing to the changes in ocis monorepo

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -11,12 +11,12 @@ concurrency:
 
 steps:
 - name: sync
-  image: xoxys/git-batch:latest
+  image: thegeeklab/git-batch:latest
   commands:
-  - git-batch
+  - git-batch -vvv
```
Comment (xoxys) [inline .drone.yml:16]: `-vvv` will switch to debug mode, `-vv` would be enough for info and warning (just as information) up to you what level to choose

File: .drone.yml [hunk 2]
```diff
@@ -11,12 +11,12 @@ concurrency:
...
 
 - name: build
-  image: owncloudci/hugo:0.69.2
+  image: owncloudci/hugo:0.71.0
```
Comment (xoxys) [inline .drone.yml:19]: Current upstream release is v0.75, maybe upgrade if you touch it anyway

**Review by xoxys — APPROVED** [review]
See comments, LGTM

File: .drone.yml (hunk 2 above)
Comment (micbar) [reply .drone.yml:19]: please not now :-) I am not sure about the implications. Need to test first.

File: .drone.yml (hunk 1 above)
Comment (micbar) [reply .drone.yml:16]: ok

Comment (micbar) [reply .drone.yml]: done

**Pull request merged**